lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -304,9 +302,8 @@ def _f(locs, labels):
                     c[self.nth_coord] = x
                     c1, c2 = data_to_axes.transform(c)
                     if (0 <= c1 <= 1 and 0 <= c2 <= 1
-                            and 0 - self.delta1
-                                <= [c1, c2][self.nth_coord]
-                                <= 1 + self.delta2):
+                            and mpl.transforms._interval_contains_close(
+                                (0, 1), [c1, c2][self.nth_coord])):
```

Comment:
I wonder what the point of this check is; if `0 <= c1 <= 1`, then `0-delta1 <= c1 <= 1+delta2` automatically.

Reply:
Agreed that this seems redundant; removed.